exercises/bob/canonical-data.json

```diff
@@ -28,6 +28,16 @@
       },
       "expected": "Whoa, chill out!"
     },
+    {
+      "uuid": "c10699d2-7a89-40c8-9cd3-ed277fe1c14a",
+      "description": "shouting multilingual gibberish",
+      "scenarios": ["unicode"],
+      "property": "response",
+      "input": {
+        "heyBob": "FCÄEÜCÖDFCẞAB"
```

Comment:
It's not clear in GitHub's fonts, but this is actually an uppercase ẞ, not a lowercase ß.